src/main/resources/templates/fragments/navbar.html

```diff
@@ -7,7 +7,13 @@
      id="navigation-example">
     <div class="container-fluid">
         <div class="navbar-wrapper">
-            <a class="navbar-brand" href="javascript:void(0)">Dashboard</a>
+<!--            <a class="navbar-brand" href="javascript:void(0)">Dashboard</a>-->
+            <span class="gruzini-navbar-text"
+                    th:text="${session.loggedUser.player.firstName + ' ' + session.loggedUser.player.lastName}"></span>
```

Comment:
A może spróbuj czegoś takiego: th:text ="${#authentication.getPrincipal().getUser().getFirstName()}" to nie będzie trzeba nawet dodawać samemu nic do sesji. Pytanie tylko czy uda się tu w thymeleafie wyciągnąć imie nazwisko i punkty

Reply:
No właśnie próbowałem tego i nie pamiętam czy w ogóle cokolwiek wyciągnęło, a na pewno nie ma dostępu do czegokolwiek więcej niż do username'a principala. Pamiętaj, że po zaciągnięcu Principala korzystaliśmy z serwisu, żeby wyciągnąć usera, on domyślnie w Principalu nie siedzi.

Reply:
okok, dlatego zaznaczyłem żeby wybadać